File: 체육복/javascript/20221103/solution.test.js
import solution from './solution';

test('도난당한 학생과 여벌 체육복이 있는 학생 구하기', () => {
  expect(solution(5, [2, 4], [1, 3, 5])).toStrictEqual(5);
  expect(solution(5, [2, 3], [3])).toStrictEqual(4);
  expect(solution(5, [2, 4], [3])).toStrictEqual(4);
  expect(solution(3, [3], [1])).toStrictEqual(2);
});

File: 이진변환반복하기/java/20221219/test/SolutionTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class SolutionTest {
    @Test
    void solution() {
        Solution solution = new Solution();

        int[] answer = solution.solution("110010101001");

        assertEquals(answer[0], 3);
        assertEquals(answer[1], 8);

        int[] answer2 = solution.solution("01110");

        assertEquals(answer2[0], 3);
        assertEquals(answer2[1], 3);

        int[] answer3 = solution.solution("1111111");

        assertEquals(answer3[0], 4);
        assertEquals(answer3[1], 1);
    }
}
